### PR TITLE
fix(styles): fixing invalid width in scss

### LINKED ISSUE #311: Unexpected width value in .user-page .provider .favicon CSS width
In [/app/styles/modules/_user_page.scss:35](https://github.com/mozilla/chronicle/blob/408fdde81b084704b5c5c82c6f9506bda7b58d1e/app/styles/modules/_user_page.scss#L35) we have the following code:
```scss
    .favicon {
      height: $half-grid;
      margin-right: 5px;
      vertical-align: -3px;
      width: 1$half-grid; /* <---- this guy! */
    }
```

Which compiles to this [de-minified] code:
```css
    .user-page .provider .favicon {
      height: 16px;
      margin-right: 5px;
      vertical-align: -3px;
      width: 1 16px; /* <---- this guy! */
    }
```

CSSLint blows up with this hawkward error:
> _**WARNING:** Expected end of value but found '16px'. Properties should be known (listed in CSS3 specification) or be a vendor-prefixed property. (known-properties) Browsers: All_

### PR DESCRIPTION
**Pull request by pdehaan**
Fixes #311

### CODE REVIEW

File: app/styles/.scss-lint.yml [hunk 1]
```diff
@@ -17,6 +17,9 @@ linters:
     enabled: true
     style: include_zero # or 'exclude_zero'
 
+  PlaceholderInExtend:
+    enabled: false
+
```
Comment (pdehaan) [inline app/styles/.scss-lint.yml]: This gets rid of the `[W] Prefer using placeholder selectors (e.g. %some-placeholder) with @extend` warnings from [**scss-lint**](https://github.com/causes/scss-lint).

**Comment by johngruen** [comment]
@pdehaan: running `scss-lint app/styles` is still warning me on nesting depth

screen shot 2015-02-09 at 6 15 16 pm

**Comment by pdehaan** [comment]
Weird, I set the `SelectorDepth` to 4 in [/app/styles/.scss-lint.yml](https://github.com/mozilla/chronicle/blob/master/app/styles/.scss-lint.yml)

```sh
$ scss-lint --version
scss-lint 0.25.1
```
(although I think the latest gem version may be 0.33.0)

This is the current output that I get:
```sh
$ scss-lint app/styles

app/styles/_breakpoints.scss:15 [W] @else should be placed on same line as previous curly brace
app/styles/_breakpoints.scss:17 [W] Prefer single quoted strings
app/styles/modules/_global_header.scss:16 [W] @include declaration should be preceded by an empty line
app/styles/modules/_global_header.scss:28 [W] @include declaration should be preceded by an empty line
app/styles/modules/_row.scss:14 [W] @include declaration should be preceded by an empty line
app/styles/modules/_row.scss:17 [W] @include declaration should be followed by an empty line
app/styles/modules/_row.scss:26 [W] @include declaration should be preceded by an empty line
app/styles/modules/_user_info.scss:4 [W] @include declaration should be followed by an empty line
app/styles/modules/_user_page.scss:112 [W] Merge rule `&.small + &.small` with rule on line 90
```

90% of those are just silly cosmetic. And I don't really like the thought of refactoring this one since I like your current organization of rules:
```
app/styles/modules/_user_page.scss:112 [W] Merge rule `&.small + &.small` with rule on line 90
```

**Comment by pdehaan** [comment]
Nevermind... It looks like you're probably using a newer version of scss-lint than I am. I upgraded to 0.33.0 and now I can see that. Looks like I may need to tweak some rules.

**Comment by pdehaan** [comment]
Current **scss-lint@0.33.0** output:
```sh
$ scss-lint app/styles

app/styles/_breakpoints.scss:15 [W] ElsePlacement: @else should be placed on same line as previous curly brace
app/styles/_breakpoints.scss:17 [W] StringQuotes: Prefer single quoted strings
app/styles/modules/_global_header.scss:16 [W] EmptyLineBetweenBlocks: @include declaration should be preceded by an empty line
app/styles/modules/_global_header.scss:28 [W] EmptyLineBetweenBlocks: @include declaration should be preceded by an empty line
app/styles/modules/_row.scss:14 [W] EmptyLineBetweenBlocks: @include declaration should be preceded by an empty line
app/styles/modules/_row.scss:17 [W] EmptyLineBetweenBlocks: @include declaration should be followed by an empty line
app/styles/modules/_row.scss:26 [W] EmptyLineBetweenBlocks: @include declaration should be preceded by an empty line
app/styles/modules/_user_info.scss:2 [W] DeclarationOrder: Rule sets should be ordered as follows: @extends, @includes without @content, properties, @includes with @content, nested rule sets
app/styles/modules/_user_info.scss:4 [W] EmptyLineBetweenBlocks: @include declaration should be followed by an empty line
app/styles/modules/_user_page.scss:112 [W] MergeableSelector: Merge rule `&.small + &.small` with rule on line 90
```

**Comment by johngruen** [comment]
@pdehaan: hmm, also running 0.33.0

```
chronicle (issue-311) scss-lint app/styles/
app/styles/_breakpoints.scss:15 [W] ElsePlacement: @else should be placed on same line as previous curly brace
app/styles/_breakpoints.scss:17 [W] StringQuotes: Prefer single quoted strings
app/styles/_layout.scss:7 [W] IdSelector: Avoid using id selectors
app/styles/_layout.scss:11 [W] IdSelector: Avoid using id selectors
app/styles/modules/_global_footer.scss:1 [W] IdSelector: Avoid using id selectors
app/styles/modules/_global_header.scss:1 [W] IdSelector: Avoid using id selectors
app/styles/modules/_global_header.scss:16 [W] EmptyLineBetweenBlocks: @include declaration should be preceded by an empty line
app/styles/modules/_global_header.scss:26 [W] IdSelector: Avoid using id selectors
app/styles/modules/_global_header.scss:28 [W] EmptyLineBetweenBlocks: @include declaration should be preceded by an empty line
app/styles/modules/_row.scss:14 [W] EmptyLineBetweenBlocks: @include declaration should be preceded by an empty line
app/styles/modules/_row.scss:17 [W] EmptyLineBetweenBlocks: @include declaration should be followed by an empty line
app/styles/modules/_row.scss:26 [W] EmptyLineBetweenBlocks: @include declaration should be preceded by an empty line
app/styles/modules/_user_info.scss:1 [W] IdSelector: Avoid using id selectors
app/styles/modules/_user_info.scss:2 [W] DeclarationOrder: Rule sets should be ordered as follows: @extends, @includes without @content, properties, @includes with @content, nested rule sets
app/styles/modules/_user_info.scss:4 [W] EmptyLineBetweenBlocks: @include declaration should be followed by an empty line
app/styles/modules/_user_info.scss:81 [W] NestingDepth: Nesting should be no greater than 3, but was 4
app/styles/modules/_user_page.scss:75 [W] NestingDepth: Nesting should be no greater than 3, but was 4
app/styles/modules/_user_page.scss:80 [W] NestingDepth: Nesting should be no greater than 3, but was 4
app/styles/modules/_user_page.scss:112 [W] MergeableSelector: Merge rule `&.small + &.small` with rule on line 90
```

**Comment by pdehaan** [comment]
Me fail the build? That's unpossible!

**Comment by pdehaan** [comment]
@johngruen You may need to repull the .scss-lint.yml config file, I turned off the `IdSelector` rule and added a `NestingDepth` rule and set it to 4 in mein latest commit.

**Comment by johngruen** [comment]
@pdehaan: hmm, okay, somehow i was behind you on this branch even though i added your fork as an origin like two minutes ago. Seeing changes now.

**Pull request merged**